packages/cli/src/routeGeneration/templates/koa.hbs

```diff
@@ -24,7 +24,7 @@ import { iocContainer } from '{{iocModule}}';
 import { IocContainer, IocContainerFactory } from '@tsoa/runtime';
 {{/if}}
 import type { Middleware } from 'koa';
-import * as KoaRouter from '@koa/router';
```

Comment:
I don't know Koa so can't test this, but I think it's covered in the tests and I assume it's better to have a consistent import usage across the code base?